bamboo-specs/bamboo.yml

```diff
@@ -0,0 +1,257 @@
+---
+version: 2
+plan:
+  project-key: CALC
+  key: CML
+  name: Calculation Module Latest
+stages:
+- Default Stage:
+    manual: false
+    final: false
+    jobs:
+    - Build and Test JDK 8
+    - Build and Test JDK 11
+    - Build and Test JDK 21
+    - Build and Test JDK 24
+    - Build and Test JDK 17
+- Deploy:
+    manual: false
+    final: false
+    jobs:
+    - Deploy to Maven
+- Release:
+    manual: true
+    final: false
+    jobs:
+    - Release to Maven
+Build and Test JDK 8:
+  key: BTJ8
+  tasks:
+  - checkout:
+      force-clean-build: 'false'
+      description: Checkout default repository
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash -eu
+
+        set -x
+
+        export IMAGE="maven:3.9.9-amazoncorretto-8"
```

Comment:
Do we still need the above line?

Reply:
Yep, it’s just to make it easier for others to find the right variable when making changes.

Reply:
When you say `might change`, many times this never happens. That is how we end up with lots of things in our code bases that are never used. But a developer thought that this might be of use in the future. I do not see us reducing the clarity of this code by using `maven:3.9.9-amazoncorretto-8`, unless we want to use it to teach others the meaning of the parameters for the docker run command. 😊